rviz_common/src/rviz_common/temp/stl_loader.cpp

```diff
@@ -80,13 +80,13 @@ bool STLLoader::load(const std::string & path)
 
   // find the file size
   fseek(input, 0, SEEK_END);
-  long fileSize = ftell(input);
+  long fileSize = ftell(input);  // NOLINT maybe use int64 instead
```

Comment:
No maybe, just say `ftell` returns long, or use `auto`.

Reply:
I will use a better description in the comment.
Using `auto` will not work due to the comparison a few lines down between `fileSize` and `num_bytes_read`.